rxbinding/src/main/java/com/jakewharton/rxbinding/view/RxView.java

```diff
@@ -181,6 +182,35 @@
   }
 
   /**
+   * Create an observable which emits on {@code view} globalLayout events. The emitted value is
+   * unspecified and should only be used as notification.
+   * <p></p>
+   * <em>Warning:</em> The created observable keeps a strong reference to {@code view}. Unsubscribe
+   * to free this reference.
+   * <p>
+   * <em>Warning:</em> The created observable uses {@link ViewTreeObserver#addOnGlobalLayoutListener} to observe
+   * globalLayouts. Multiple observables can be used for a view at a time.
+   */
+  @CheckResult @NonNull
+  public static Observable<Object> globalLayouts(@NonNull View view) {
+    return Observable.create(new ViewTreeObserverGlobalLayoutOnSubscribe(view));
+  }
+
+  /**
+   * Create an observable of {@link ViewTreeObserverGlobalLayoutEvent} for globalLayouts on {@code view}.
+   * <p>
+   * <em>Warning:</em> The created observable keeps a strong reference to {@code view}. Unsubscribe
+   * to free this reference.
+   * <p>
+   * <em>Warning:</em> The created observable uses {@link ViewTreeObserver#addOnGlobalLayoutListener} to observe
+   * globalLayouts. Multiple observables can be used for a view at a time.
+   */
+  @CheckResult @NonNull
+  public static Observable<ViewTreeObserverGlobalLayoutEvent> globalLayoutEvents(@NonNull View view) {
```

Comment:
Let's delete this per https://github.com/JakeWharton/RxBinding/pull/121/files#r40500931